tests/virt_autotest/kubevirt_tests_server.pm

```diff
@@ -219,6 +234,12 @@ sub install_kubevirt_packages {
     zypper_call("--gpg-auto-import-keys ref");
 
     my $virt_manifests = 'containerized-data-importer-manifests kubevirt-manifests kubevirt-virtctl';
+    my $virt_tests = 'kubevirt-tests';
+    my $container_tag = get_var("CONTAINER_TAG");
```

Comment:
There is get_required_var("CONTAINER_TAG") on [this line](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/d59d28ba71b466266ff35b3957d72b2f13202af3/tests/virt_autotest/kubevirt_tests_server.pm#L536). Is it necessary(get_required_var) or not(get_var)? If necessary, would you please explain a little bit why? I remember in my tryings, after installing the pkgs, it is optional to indicate it, although it is necessary to indicate PREVIOUS_UTILITY_CONTAINER_TAG...

Reply:
Thanks for the reminder, this is a required option. Refer to https://bugzilla.suse.com/show_bug.cgi?id=1210542#c2

Reply:
I see. Is there any relationship between this CONTAINER_TAG and virt-* containers tag? How are we using it? Do we build the containers in our own registry each time for different kubevirt versions? Or we just pull fixed version that is built previously?